### ISSUE TITLE
Typo in Training-PPO.md?

### ISSUE THREAD

**Issue by wil3** [Unity-Technologies/ml-agents#2044]
For [Number of Epochs](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-PPO.md#number-of-epochs), wouldn't **increasing** the number of epochs result in slower learning, not **decreasing**?

Excellent doc though, very helpful.

**Comment by xiaomaogy** [comment]
Increasing the number of epoch will actually speed up the training, at the cost of increasing the training instability. Basically we collect some data on the Unity side, then pass the data to the python side to update the model. We can choose to update the model with this data for (number of epoch) times (by running gradient descent like update). If we update the model more frequently without collecting the new data, the model should learn faster, but maybe the model update is no longer effective because we are updating it based on old data information, which leads to training instability. 

Hopefully this explanation make sense. @wil3

**Comment by wil3** [comment]
Hi @xiaomaogy thanks for the response. Your explanation makes sense, I think I was confusing myself with the learning time verse wall clock. Would it be fair to say that the lapse time on training would increase though because you've now introduced 'number of epoch' more iterations. I guess it doesn't really matter though if your time is just relative to when the agent is sufficiently trained?

**Comment by xiaomaogy** [comment]
Yes the lapse time on training would increase, but the wall clock training time would decrease if the instability issue doesn't break the training.

**Comment by xiaomaogy** [comment]
Thank you for the discussion. We are closing this issue due to inactivity. Feel free to reopen it if you’d like to continue the discussion though.
